### ISSUE TITLE
Web/mobile: Conflicts folder (when selected) has low contrast with dark theme

### ISSUE THREAD

**Issue by personalizedrefrigerator** [laurent22/joplin#13437]
### Operating system

Linux

### Joplin version

38ea92f

### Desktop version info

Joplin Mobile 3.5.0 (prod, web)

Device: web
Client ID: fd8d7b4bbfe94caab60e3f83b0006adf
Sync Version: 3
Profile Version: 48
Keychain Supported: No
Alternative instance ID: -

Revision: 38ea92f (dev)

Extra Markdown editor settings: 1.11.0
Joplin debug tool: 0.3.0
Work tracker: 0.0.1

User agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/141.0.0.0 Safari/537.36
FTS enabled: 1
Hermes enabled: 0

### Current behaviour

Currently, the conflicts label has a contrast of [1.54:1](https://webaim.org/resources/contrastchecker/?fcolor=FF0000&bcolor=616161) when selected and the default dark theme is enabled.

The label's contrast is higher when unselected ([4.08:1](https://webaim.org/resources/contrastchecker/?fcolor=FF0000&bcolor=1D2024)) but also low.

<details><summary>Screenshots</summary>

<img width="702" height="424" alt="screenshot: Chrome contrast checker identifies the selected label as having a 1.54:1 contrast ratio" src="https://github.com/user-attachments/assets/e90aa979-257f-4d8e-aaa7-d2f2c4702bb6" />

<img width="702" height="424" alt="screenshot: Chrome contrast checker identifies the unselected label as having a 4.08:1 contrast ratio" src="https://github.com/user-attachments/assets/c18a2bc2-4204-492c-8f72-dec2562e6077" />

</details>

### Expected behaviour

As per [WCAG 2.2 SC 1.4.3](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html), the conflicts label should have a contrast of at least 4.5:1.

**Comment by mrjo118** [comment]
Is the issue that highlighting the text reduces opacity / lightness of the text? Presumably if the text actually showed through as #FF0000, then the contrast would be much greater

**Comment by personalizedrefrigerator** [comment]
The issue seems to be that `#FF0000` has low contrast with the highlight and background colors used in dark mode (`#616161` and `#1D2024`). 

See the links to the WebAim contrast checker:
> - [1.54:1](https://webaim.org/resources/contrastchecker/?fcolor=FF0000&bcolor=616161)
> - [4.08:1](https://webaim.org/resources/contrastchecker/?fcolor=FF0000&bcolor=1D2024)

The fix will likely involve using a different foreground color when the app is in dark mode.

Thank you for looking into this!

**Comment by personalizedrefrigerator** [comment]
Re-opening as partially resolved.
